nab-starter/src/main/java/ru/hh/nab/starter/NabLogbackBaseConfigurator.java

```diff
@@ -40,7 +41,11 @@ public final void configure(LoggingContextWrapper context) {
 
     HhMultiAppender service = createAppender(context, "service", () -> new HhMultiAppender(true));
 
-    HhMultiAppender requests = createAppender(context, "requests", () -> new HhMultiAppender(true));
+    HhMultiAppender requests = createAppender(context, "requests", () -> {
+      var multiAppender = new HhMultiAppender(true);
+      multiAppender.setLayout(new StructuredRequestJsonLayout());
```

Comment:
https://github.com/hhru/nuts-and-bolts/blob/99a5f9e682737490fc66f4e8503e9f7c6046d3c4/nab-starter/src/main/resources/nab-shared-logback.xml#L33